Code/Framework/AzCore/Tests/Date/DateFormatTests.cpp

```diff
@@ -0,0 +1,365 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Date/DateFormat.h>
+#include <AzCore/UnitTest/TestTypes.h>
+
+namespace UnitTest
+{
+    //////////////////////////////////////////////////////////////////////////
+    // Fixtures
+
+    // Fixture for non-typed tests
+    class DateFormatTest
+        : public ScopedAllocatorSetupFixture
+    {
+    };
+
+    namespace Iso8601Timestamp
+    {
+        TEST_F(DateFormatTest, DateFormat_ExtendedFormatDate_Succeeds)
+        {
+            using namespace AZStd::chrono;
+            using namespace AZStd::chrono_literals;
```

Comment:
suggestion: Instead of `using namespace` here you could instead use `using AZStd::chrono::year_month_day;` and `using AZStd::chrono::operator""_y;` etc.. (a little more typing but avoids any weird name lookup rules with `using namespace`

Reply:
Well this `using namespace` is localized to the test function which is only using chrono function and not mixing with other libraries, so it should cause any issues.

Reply:
Unfortunately that isn't the case, `using namespace` will inject these names into the global scope and can lead to really weird problems to do with name look-up rules (see this article for more details - https://abseil.io/tips/153)

Reply:
These names are not getting injected into the global cope. Because the nearest namespace is `UnitTest::Iso8601Timestamp`